### ISSUE TITLE
Timeout on map server

### ISSUE THREAD

**Issue by Fruitard** [OpenKore/openkore#3758]
* **Openkore version git**: 
OpenKore what-will-become-2.1 
* **Server**: 
Private Server(serverType kRO_RagexeRE_2013_12_23c)
* **Bug Report / Feature Request**:
Timeout on map server after connecting to character
* **Summary**:
It was all working fine but after server decided to change to another host, now i am stuck at Timeout on map server
![image](https://github.com/OpenKore/openkore/assets/31286810/3af959ea-a86b-448d-9fab-76c5ab60ce7a)

This is my server config

> port 6900
> master_version 1
> version 45
> charBlockSize 144
> serverType kRO_RagexeRE_2013_12_23c
> addTableFolders kRO/RagexeRE_2013_12_23c;translated;translated/kRO_english;kRO
> private 1
> pauseCharLogin 1
> sendCryptKeys 0x631c511c, 0x111c111c, 0x111c111c
> 

Some solution which i have tried : 

1. Changing sendCryptKeys 
2. Removing of sendCryptKeys 
3. Extracting new recvpacket

**Comment by alisonrag** [comment]
key order in openkore is 1,3,2

**Comment by Fruitard** [comment]
> key order in openkore is 1,3,2
> 
> edit: sorry not checked 2 and 3 are the same, did u checked the client date?
> 
> if they are using the default keys and the keys changed should mean they changed the client

yes the keys for both 2 and 3 is the same. i have downloaded the new client as well but it shows the same key

**Comment by Fruitard** [comment]
> key order in openkore is 1,3,2
> 
> edit: sorry not checked 2 and 3 are the same, did u checked the client date?
> 
> if they are using the default keys and the keys changed should mean they changed the client

I have tried download their new installer and extract the recvpacket. but the keys are still the same

**Comment by alisonrag** [comment]
try to use microsoft network monitor or wireshark to check the traffic data when login into map-server, post here to check if is encrypted or not

**Comment by Fruitard** [comment]
> try to use microsoft network monitor or wireshark to check the traffic data when login into map-server, post here to check if is encrypted or not

![image](https://github.com/OpenKore/openkore/assets/31286810/538249bd-9c7d-4c70-aa07-de7651756e9c)
is it this?

**Comment by Fruitard** [comment]
> > try to use microsoft network monitor or wireshark to check the traffic data when login into map-server, post here to check if is encrypted or not
> 
> ![image](https://user-images.githubusercontent.com/31286810/241613371-538249bd-9c7d-4c70-aa07-de7651756e9c.png) is it this? this was using wireshark
> 
> edit : since their changing of host to another location, the bot is not able to connect pass their map server.

hi @alisonrag is this the correct data? i am not familiar with traffic data so i am not sure if i am posted the correct information

**Comment by alisonrag** [comment]
to me this is just a hex array, maybe u can save the wireshark capture. start the capture when select server and stop after enter in game and walk

**Comment by Fruitard** [comment]
> > to me this is just a hex array, maybe u can save the wireshark capture. start the capture when select server and stop after enter in game and walk
> 
> hi you mean this? i have to upload to wetransfer as i cant upload wireshark file here
> 
> https://we.tl/t-ujhlmAlRFf

hi @alisonrag, is this the correct file that you were asking for? I have tried other ways to see if i am able to get passed map server, but it still doesn't work

**Comment by Fruitard** [comment]
> alison#5129

thank you, i have added you in discord
